utils/build/docker/dotnet/poc.Dockerfile

```diff
@@ -1,19 +1,27 @@
 FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /app
 
-RUN apt-get update && apt-get install dos2unix
+# required by 'install_ddtrace.sh' to call 'strings' command
+RUN apt-get update && apt-get install -y binutils
```

Comment:
[Note to reviewers]
- removed `dos2unix`, not needed anymore
- add `binutils`, required by `install_ddtrace.sh`